Transcriptome: Set of all RNA molecules: mRNA+tRNA+rRNA  + other non coding RNA

Difference between exome and Transcriptome: Transcriptome limited to specific cell population

Exomes: Protein coding regions, collection of exons
Transcriptome also captures the totality of RNA product, hence would possibly also include spliced products

If you were to profile Exome and transcriptome together, it is possible that
their content would not exactly be the same. Besides including RNA editing events,
the transcriptome would also icnlude spliced variants.

RNA editing: Certian enzymes modify mRNA post transcriptionally.
Example: 
ADAR(Adenosine deaminases that act on RNA) convert A->I->G
APOBEC(Apolipoprotein B mRNA editing enzymes) convert C->U
c->U are rarer than A->G
See: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3204392/

Exome sequencing: Sequencing of all the exons(DNA based)
RNA-seq: Sequencing of the transcriptome: Represents a quasi-static 

It should thus be possible to identify coding SNPs using either of these technologies, however
RNA-seq results are reported to be biased: http://genomebiology.com/content/pdf/gb-2010-11-5-r57.pdf

:wq

Number of TCells we make: 2.5x10^7
The receptor chains are encoded by different gene segments. So the genes can be much 
less in number and still produce more TCRs/BCRs

Why SingleCell RNA-seq:?



RNA-seq:
counts are simply the number of reads overlapping a given feature such as a gene.

:q
